docs/tutorials/hpa-recommendation.zh.md

```diff
@@ -0,0 +1,219 @@
+# 弹性推荐
+
+通过弹性推荐，你可以发现集群中适合弹性的资源，并使用 Crane 推荐的弹性配置创建自动弹性器: EffectiveHPA
+
+## 创建弹性分析
+
+创建一个**弹性分析** `Analytics`，这里我们通过实例 deployment: `nginx` 作为一个例子
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/gocrane/crane/main/examples/analytics/nginx-deployment.yaml
+kubectl apply -f https://raw.githubusercontent.com/gocrane/crane/main/examples/analytics/analytics-hpa.yaml
+kubectl get analytics
+```
+
+```yaml title="analytics-hpa.yaml"
+apiVersion: analysis.crane.io/v1alpha1
+kind: Analytics
+metadata:
+  name: nginx-hpa
+spec:
+  type: HPA                        # This can only be "Resource" or "HPA".
+  completionStrategy:
+    completionStrategyType: Periodical  # This can only be "Once" or "Periodical".
+    periodSeconds: 600                  # analytics selected resources every 10 minutes
+  resourceSelectors:                    # defines all the resources to be select with
+    - kind: Deployment
+      apiVersion: apps/v1
+      name: nginx-deployment
+  config:                               # defines all the configuration for this analytics
+    ehpa.deployment-min-replicas: "1"
+    ehpa.fluctuation-threshold: "0"
+    ehpa.min-cpu-usage-threshold: "0"
+```
+
+结果如下:
+
+```bash
+NAME        AGE
+nginx-hpa   16m
+```
+
+查看 Analytics 的 Status，通过 status.recommendations[0].name 得到 Recommendation 的 name:
+
+```bash
+kubectl get analytics nginx-hpa -o yaml
+```
+
+结果如下:
+
+```yaml
```

Comment:
Make `name: nginx-hpa-hpa-cd86s` hightlight.
Easy for people to follow the guides.

```
The recommendation name presents on `status.recommendations[0].name`. Then you can get recommendation detail by running:
```

```suggestion
```yaml hl_lines="32"
```